Translation corrections (#82)
07th-mod/higurashi-console-arcs: Update/some_005.txt

//rやっぱり私、幸せだなぁ…！！
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "やっぱり私、幸せだなぁ…！！",
			NULL, "I really am happy...!", Line_Normal);
	ClearMessage();

//r好きな人に告白されて、心から親友と呼べる友達にこの恋を応援されて…。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "好きな人に告白されて、心から親友と呼べる友達にこの恋を応援されて…。",

> 			NULL, "The boy I like confessed to me, and the one I considered my dear friend is rooting for our love to succeed.", Line_Normal);

Reviewer: ```suggestion
			NULL, "The boy I like confessed to me, and someone I consider my dear friend is rooting for our love to succeed...", Line_Normal);
```

			NULL, "\"Kyah!", Line_WaitForInput);
	ModPlayVoiceLS(4, 38, "ps3/s04/38/310400134", 256, TRUE);
	OutputLine(NULL, "　なぁ～んだ、千紗登か。",
			NULL, " Oh, it's just you, Chisato.", Line_WaitForInput);
	ModPlayVoiceLS(4, 38, "ps3/s04/38/310400135", 256, TRUE);
	OutputLine(NULL, "びっくりしたぁ～！！」",
			NULL, " You surprised me~!!\"", Line_Normal);
	ClearMessage();

	FadeAllBustshots( 500, TRUE );

//r私の脇をするりと抜けて、千紗登ちゃんは珠子ちゃんに駆け寄って、その肩をぽんっと叩く。珠子ちゃんは心底びっくりしたように、胸に手をあてた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "私の脇をするりと抜けて、千紗登ちゃん は珠子ちゃんに駆け寄って、その肩をぽんっと叩く。珠子ちゃんは心底びっくりしたように、胸に手をあてた。",

> 			NULL, "Chisato-chan rushed over to Tamako-chan to pat her on the shoulder. Tamako-chan put a hand on her chest as thought she'd been deeply surprised.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Chisato-chan rushed over to Tamako-chan to pat her on the shoulder. Tamako-chan put a hand on her chest as though she'd been deeply surprised.", Line_Normal);
```

//r無機質で冷たく、…耳に再び蘇る、さっきの酷薄な口調。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "無機質で冷たく、…耳に再び蘇る、さっきの酷薄な口調。",
			NULL, "It sounded inorganic and cold... That same cruel tone from before was echoing in my ear.", Line_Normal);
	ClearMessage();

//rあれ…？　千紗登ちゃん…？　すごく冷たい言い方。…あれ？
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "あれ…？　千紗登ちゃん…？　すごく冷たい言い方。…あれ？",

> 			NULL, "Huh...? Chisato-chan? Why do sound so cold. ...What?", Line_Normal);

Reviewer: ```suggestion
			NULL, "Huh...? Chisato-chan? Why do you sound so cold? ...What?", Line_Normal);
```

			NULL, " Well...?", Line_WaitForInput);
	ModPlayVoiceLS(4, 38, "ps3/s04/38/310400151", 256, TRUE);
	OutputLine(NULL, "　…そのことと、待ち合わせに遅くなったことと…、何か関係があるの？」",
			NULL, " ...Did that have anything to do... with why you arrived late to our meeting?\"", Line_Normal);
	ClearMessage();

//r優しく包み込むような声で、珠子ちゃんが問いかける。…その目は、ちっとも笑ってもいなければ、私の幸福など微塵も願っていないような、冷たい光を宿していた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "優しく包み込むような声で、珠子ちゃんが問いかける。…その目は、ちっとも笑ってもいなければ、私の幸福など微塵も願っていないような、冷たい光を宿していた。",

> 			NULL, "Tamako-chan asked that with a voice was wrapped in kindness. ...But not only did her eyes lack a hint of mirth, they also dwelled with a cold light that didn't wish a single speck of happiness for me.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Tamako-chan asked that with a voice wrapped in kindness. ...But not only did her eyes lack a hint of mirth, in them also dwelled a cold light that didn't wish a single speck of happiness for me.", Line_Normal);
```

//r咄嗟にでたらめを口にしてしまう。だけどそれはすぐにバレる嘘だと悟って、私は後悔した。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "咄嗟にでたらめを口にしてしまう。だけどそれはすぐにバレる嘘だと悟って、私は後悔した。",

> 			NULL, "I blurted out the first nonsense that came to mind. But I regretted it as soon as I realized she'd immediately see through that lie.", Line_Normal);

Reviewer: ```suggestion
			NULL, "I blurted out the first nonsense that came to mind. But I regretted it as soon as I realized she'd immediately seen through that lie.", Line_Normal);
```
